clients/client-accessanalyzer/package.json

```diff
@@ -95,6 +95,10 @@
   "react-native": {
     "./dist-es/runtimeConfig": "./dist-es/runtimeConfig.native"
   },
+  "typedoc": {
```

Comment:
The entrypoint and readmeFile is defined in typedoc.json

Is this config in package.json redundnant?

Reply:
this here is actually needed for the client SDK itself because they are generated in isolation and not in the context of the Yarn workspaces